<a href="https://colab.research.google.com/github/gayathripriya01/Basic-ML/blob/main/MY_FIRST_NOTEBOOK.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

* EVEN OR ODD

In [ ]:
n=int(input())
if( n%2 == 0 ): 
   print("even") 
else: 
   print("odd")

6
even


Explanation:
 
1.This program is used to find whether the number is even or odd. 
 
2.First let us consider an input n. Here 'int' represents the input is an integer. 
 
3.The condition for an integer to be an even number is "It must be divisible by 2"and the condition for the odd number is "It is not divisible by 2".  
 
4.So,I have taken the condition that is if the number is divisible by two then the output will be even and if the number is not divisible by two then the output will be odd. 
 
5.In this program I have used 'if-else' condition . 
 
6."print" is used give the output. 
 
7.Here I have given input as 6 .Since 6 is even number the output printed is even.

* FACTORIAL OF A GIVEN NUMBER

In [12]:
n=int(input()) 
factorial=1 
for i in range(1,n+1): 
 factorial=factorial*i
print(factorial)

6
720


Explanation:  
 
1.This program is used to find the factorial of the given number. 
 
2.I have used "for loop " in this program . 
 
3.I have taken the required input as 'n'.Since the input is an integer, 'int' is used to represent it.  
 
4.I have given the factorial value as 1.And 1 is stored in it.
 
5.I have taken the range of i from 1 to (n+1) . 
  
6.Each value of i is multiplied with the factorial which will be stored in factorial.  
 
7.Now use the print which is used to print the factorial of the given number after the for loop is ended. 
 
8.Here I have given the input as 6 .The factorial of six is '720'.Hence the output printed is 720.

 * SUM OF FIRST 'n' NATURAL NUMBERS

In [16]:
n=int(input()) 
sum=0 
for i in range(1,n+1):
 sum=sum+i  
 i=i+1
print(sum)

6
21


Explanation: 
 
1.This program is used to find the sum of n natural numbers.
 
2."for loop" is used in this program. 
 
3.'n' is the input here and 'int' represents that input is an integer.  
 
4.Here for loop is runned in the range of i from 1 to (n+1) .
 
5.sum value is assigned as zero. 
 
6.Every value of i is multiplied with the sum .And is stored in sum. 
 
7."print" is used to print the output. 
 
8.Here I have given the input as 6. Since the Sum of first 6 numbers is 21.So the output printed is 21.